src/genie/libs/parser/iosxe/show_ap.py

```diff
@@ -295,3 +295,782 @@ def cli(self, output=None):
+        #   DCA Sensitivity Level                      : MEDIUM : 15 dB
+        #   DCA 802.11n/ac Channel Width               : 80 MHz
+        #   DBS Max Channel Width                      : 80 MHz
+        #   DCA Minimum Energy Limit                   : -95 dBm
+        #   Channel Energy Levels
+        #     Minimum                                  : -94 dBm
+        #     Average                                  : -82 dBm
+        #     Maximum                                  : -81 dBm
+        #   Channel Dwell Times
+        #     Minimum                                  : 4 hours 9 minutes 54 seconds
+        #     Average                                  : 4 hours 24 minutes 54 seconds
+        #     Maximum                                  : 4 hours 26 minutes 35 seconds
+        #   802.11a 5 GHz Auto-RF Channel List
+        #     Allowed Channel List                     : 36,40,44,48,52,56,60,64,100,104,108,112,116,120,124,128,132,136,140,144,149,153,157,161
+        #     Unused Channel List                      : 165
+
+        lead_auto_chan_assn_capture = re.compile(r"^Leader\s+Automatic\s+Channel\s+Assignment$")
+        #   Channel Assignment Mode                    : AUTO
+        chan_assn_mode_capture = re.compile(r"^\s+Channel\s+Assignment\s+Mode\s+:\s+(?P<chan_assn_mode>\S+)")
+        #   Channel Update Interval                    : 12 Hours
+        chan_update_int_capture = re.compile(r"^\s+Channel\s+Update\s+Interval\s+:\s+(?P<chan_upd_int>\d+)\s+Hours$")
+        #   Anchor time (Hour of the day)              : 7
+        anchor_time_capture = re.compile(
+            r"^\s+Anchor\s+time\s+\(Hour\s+of\s+the\s+day\)\s+:\s+(?P<anchor_time_hour>\d+)$")
+        #   Channel Update Contribution
+        chan_upd_cont_capture = re.compile(r"^\s+Channel\s+Update\s+Contribution$")
+        #     Noise                                    : Enable
+        channel_noise_capture = re.compile(r"^\s+Noise\s+:\s+(?P<channel_noise>(Enable|Disable))$")
+        #     Interference                             : Enable
+        channel_interference_capture = re.compile(r"^\s+Interference\s+:\s+(?P<channel_interference>(Enable|Disable))$")
+        #     Load                                     : Disable
+        channel_load_capture = re.compile(r"^\s+Load\s+:\s+(?P<channel_load>(Enable|Disable))$")
+        #     Device Aware                             : Disable
+        device_aware_capture = re.compile(r"^\s+Device\s+Aware\s+:\s+(?P<device_aware>(Enable|Disable))$")
+        #   CleanAir Event-driven RRM option           : Disabled
+        clean_air_capture = re.compile(
+            r"^\s+CleanAir\s+Event-driven\s+RRM\s+option\s+:\s+(?P<clean_air>(Enabled|Disabled))$")
+        #   Channel Assignment Leader                  : sj-00a-ewlc1 (10.7.5.133)
+        chan_assn_leader_capture = re.compile(
+            r"^\s+Channel\s+Assignment\s+Leader\s+:\s+(?P<wlc_leader_name>\S+)\s+(?P<wlc_leader_ip>\(\d+\.\d+\.\d+\.\d+\))$")
```

Comment:
`r"^\s+Channel\s+Assignment\s+Leader\s+:\s+(?P<wlc_leader_name>\S+)\s+(?P<wlc_leader_ip>\([\d\.]+\))$"`

Also, you can capture the Ip address only without the brackets so it is easily put into the returned dictionary